src/applications/vaos/services/appointment/transformers.js

```diff
@@ -299,7 +299,7 @@ function getCommunityCareData(appt) {
         ? {
             firstName: appt.name?.firstName,
             lastName: appt.name?.lastName,
-            providerName: appt.name
+            providerName: appt.name?.lastName
```

Comment:
Will `lastName` always have a value if `name` isn't an empty object? If so, then do we need to check for `firstName` and `lastName` values like we do below?

I'm wondering if it's safer to check if `appt.name` isn't an empty object if we believe that either `firstName` or `lastName` could be missing values.

Reply:
I don't know exactly what comes back here. It's hard to test, since we have to create CC appointments to do so. I think we actually get an object with the first and last name set to empty strings. My reasoning was that a last name being present would be enough data that we'd want to show that and any other shape of the data can fall back to the practice name.